### ISSUE TITLE
Option for custom wallpapers

### ISSUE THREAD

**Issue by meisme-dev** [cutefishos/settings#49]
Let users add their own wallpapers

**Comment by DanielVenturini** [comment]
Yep, that would be a very nice feature. I tried to find a way to put my own wallpapers, but it's not available yet.

**Comment by terminal-cs** [comment]
for now you can right click on an image in the cutefish file manager to set a wallpaper, but it would still be nice to have an option for this in settings, a "Browse" button would be nice.

**Comment by MatheusAznavour** [comment]
Also you can move your image and choose in the settings. Heres the path `usr/share/backgrounds/cutefishos/`
